pkg/storkctl/migrationschedule.go

```diff
@@ -43,22 +52,79 @@ func newCreateMigrationScheduleCommand(cmdFactory Factory, ioStreams genericclio
 				util.CheckErr(fmt.Errorf("ClusterPair name needs to be provided for migration schedule"))
 				return
 			}
+
+			//Identify whether it is a Sync-Dr or Async-Dr use-case from the ClusterPair spec
+			migrationScheduleNs := cmdFactory.GetNamespace()
+			clusterPairObj, err := storkops.Instance().GetClusterPair(clusterPair, migrationScheduleNs)
+			if err != nil {
+				util.CheckErr(fmt.Errorf("unable to find the cluster pair in the given namespace"))
+				return
+			}
+			isSyncDr := false
+			//clusterPairObj.Spec.Options map is empty for syncDr and non-empty for asyncDR
+			if len(clusterPairObj.Spec.Options) == 0 {
```

Comment:
We should check for the storage and scheduler status of the clusterpair before deciding if it is syncDR. It can be a miss also from the customer.

Reply:
After offline discussion with @diptiranjanpx have modified the error message a bit.
includeVolumes = false is the default and only allowed value in 2 scenarios ->
1. it is a sync-dr usecase, hence no need for volume migration as it is a single stretched cluster from storage perspective.
2. while creating the clusterpair, storage options were not provided, hence storage status is NotProvided and volume migration is not possible.